file_formats/ie_formats/cre_v1.htm

```diff
@@ -385,96 +385,123 @@
           <tr>
             <td>0x0071</td>
             <td>1 (byte)</td>
-              <td>(unused in BG2) Spears proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
+            <td>(BG1) Spears proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
           </tr>
           <tr>
             <td>0x0072</td>
             <td>1 (byte)</td>
-            <td>(unused in BG2) Blunt proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
+            <td>(BG1) Blunt proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
           </tr>
           <tr>
             <td>0x0073</td>
             <td>1 (byte)</td>
-            <td>(unused in BG2) Spiked proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
+            <td>(BG1) Spiked proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
           </tr>
           <tr>
             <td>0x0074</td>
             <td>1 (byte)</td>
-            <td>(unused in BG2) Axe proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
+            <td>(BG1) Axe proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
           </tr>
           <tr>
             <td>0x0075</td>
             <td>1 (byte)</td>
-            <td>(unused in BG2) Missile proficiency (Proficiencies maybe be packed into 3-bit chunks for the primary and secondary classes)</td>
+            <td>(BG1) Missile proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
           </tr>
           <tr>
             <td>0x0076</td>
-            <td>5 (bytes)</td>
-            <td><span class="unknown">Unused</span></td>
+            <td>1 (byte)</td>
+            <td>(BG1) Unknown proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
+          </tr>
+          <tr>
+            <td>0x0077</td>
+            <td>1 (byte)</td>
+            <td>(BG1) Unknown proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
+          </tr>
+          <tr>
+            <td>0x0078</td>
+            <td>1 (byte)</td>
+            <td>(BG1) Unknown proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
+          </tr>
+          <tr>
+            <td>0x0079</td>
+            <td>1 (byte)</td>
+            <td>(BG1) Unknown proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>
+          </tr>
+          <tr>
+            <td>0x007a</td>
+            <td>1 (byte)</td>
+            <td>(BG1) Unknown proficiency (Proficiencies are packed into 3-bit chunks for primary and secondary classes)</td>

```

Comment:
all these are available in bg2 too. This is the EXTRAPROFICIENCY20 stuff, if I wasn't clear enough. Also reserved in https://gibberlings3.github.io/iesdp/files/ids/bg2/stats.htm

Reply:
Ah, okay. I assumed it to be BG1-only because of the original remark "unused in BG2" in front of all the descriptions fields.

So if I understand it right, BG1 uses values at offsets 0x6e .. 0x75 for regular proficiencies. BG2 (and in parts Enhanced Editions as well) may use these fields in addition to offsets 0x76 .. 0x81 for extra proficiencies?

Reply:
Also probably bg1. I think bg2 stored proficiencies differently, so the whole block remained unused.

Reply:
Yeah, BG2 (and EE) uses effects instead.

So, to improve clarity I could change field descriptions to something like this (offset 0x6e):
```
BG1: Large swords proficiency
Other games: Unknown proficiency
Note: Proficiencies are packed into 3-bit chunks for primary and secondary classes.
```

Reply:
I think Unused proficiency would be clearer — there's nothing more to discover here.